backend/internal/oauth/oauth2/token/tokenhandler.go

```diff
@@ -213,5 +234,5 @@ func (th *TokenHandler) HandleTokenRequest(w http.ResponseWriter, r *http.Reques
 		return
 	}
 	// Log the token response.
-	logger.Info("Token response sent", log.String("client_id", clientID))
+	logger.Info("Token response sent", log.String("client_id", clientID), log.String("grant_type", grantType))
```

Comment:
shall we make this debug?

Reply:
Fixed with a6832baeaae66e0b2d0eb0a4feef841aa13c2acd